workspace/lib/jenkinshelper/jenkins.rb

```diff
@@ -0,0 +1,45 @@
+require 'uri'
+module JenkinsHelper
+  module Jenkins
+    def form_encode(string)
+      URI.encode_www_form_component string.encode(Encoding::UTF_8)
+    end
+
+    # Encode a string for use in the hiearchical part of an URL
+    def path_encode(path)
+      URI.escape(path.encode(Encoding::UTF_8))
+    end
+
+    def client
+      JenkinsApi::Client.new(
+      server_ip: config.ip,
+      username: config.username,
+      password: config.password)
+    end
```

Comment:
Please align it as following format
   JenkinsApi::Client.new(
      server_ip: config.ip,
      username: config.username,
      password: config.password
   )